.travis.yml

```diff
@@ -26,23 +26,23 @@ cache:
     - $HOME/.composer/cache/files
 
 env:
-  matrix: SYMFONY_VERSION=4.0.*
+  matrix: SYMFONY_VERSION=4.2.*
   global:
-    - SYMFONY_DEPRECATIONS_HELPER="/.*each.*/"
+    - SYMFONY_DEPRECATIONS_HELPER="weak"
     - SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml"
-    - SYMFONY_PHPUNIT_VERSION=6
-    - TEST_INSTALLATION=false
+    - KERNEL_CLASS=Symfony\Cmf\Bundle\RoutingBundle\Tests\Fixtures\App\Kernel
+    - SYMFONY_PHPUNIT_VERSION=5.7.26
 
 matrix:
   include:
-    - php: 7.2
-      env: SYMFONY_VERSION=4.0.*
-    - php: 7.1
-      env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=weak
-    - php: 7.2
-      env: SYMFONY_VERSION=3.3.*
-    - php: 7.2
-      env: SYMFONY_VERSION=3.4.*
+    - php: "7.3"
+      env: STABILITY="dev" SYMFONY_VERSION="4.2.*"
+    - php: "7.2"
+      env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY_VERSION="3.4.*" SYMFONY_DEPRECATIONS_HELPER="weak"
```

Comment:
are you dropping symfony 2.8 support? i think it makes sense, 2.8 is really ancient. only a bit surprised to have that as part of a general devkit update.... we might bump the minor version for it and add a note in the changelog though.

Reply:
no, i created 2.1 branch for latest stable, master goes towards 3.0-dev now.

Reply:
and yes, we do have the controll over symfony and php version out of our dev-kit

Reply:
why do you want to start a version 3, are there big BC breaking changes coming up?

if you have them, it makes sense - though i have to say upfront that i won't have much time to invest into another large-scale effort into the cmf. i would have left things and reviewed contributions and do the most important things like the effort i did over this weekend to make the tests run again.

Reply:
no worries about the time. As long as i got those initial and general hints as here i am happy.
To the BC-Breaks:
- remove SF 2.8 Support
- Use PHPUnit >=6
- SonataIntegration has to switch on FosCkEditor to get SF4 support at all -> changes in naming of our config

At the end there are no real changes in our API, but we drop some dependencies. That was the reason to move into new major. The only thinkg that is growing atm from outer persons is ORM support.

Reply:
dropping support for old versions is no BC break. composer.json will prevent from installing an impossible combination. routing does not provide phpunit related things to the outside. and the editor is only in the sonata integration.

i would prefer to do a new minor version for the phpunit and change of supported symfony versions. we could do a new major for testing and sonata integration only. that would make more sense to me. and sonata integration is anyway on a different version number than the rest, and testing used only internally.

Reply:
ok, that makes things easier and faster